lib/flex-lg.css

```diff
@@ -256,5 +274,50 @@
     margin-bottom: auto !important;
   }
 
-}
+  /**
+   * Basis
+   */
+
+  .u-lg-flexBasisAuto {
+    flex-basis: auto !important;
+  }
+
+  .u-lg-flexBasisZero {
+    flex-basis: 0% !important;
```

Comment:
Hmm, perhaps not. It's needed to fix IE10 but only when using the `flex` shorthand. I'll take it away and browser test.